src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --main-color: #00acdc;
+}
+
+.logo_img {
+  display: block;
+}
+
+.nav_link {
+  text-decoration: none;
+  position: relative;
+  color: black;
+  text-transform: uppercase;
+
+  /* height: 15px; */
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 23px 0;
```

Comment:
This is a code quality issue. It's not good to use both top and bottom margins. According to the checklist, you should be consistent with your vertical margins. It's better to add only top or only bottom margin, but not both.